deploy/osd-strimzi-operator/05-role.yaml

```diff
@@ -0,0 +1,76 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: dedicated-admins-openshift-strimzi
+  namespace: openshift-strimzi
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - namespaces
+  - pods
+  - pods/log
+  - events
+  verbs:
+  - list
+  - get
+  - watch
+- apiGroups:
+  - logging.openshift.io
+  resources:
+  - clusterloggings
```

Comment:
`ClusterLogging` CR should not be allowed in this namespace.

Reply:
We grant that on codeready workspaces NS, I understood it was there to allow customers to manage logging of those components: https://github.com/openshift/managed-cluster-config/blob/master/deploy/osd-codeready-workspaces/05-role.yaml

Reply:
If we give it to crw we should remove it.  This CR will stand up a whole new logging stack.  Probably came with copying perms for the `openshift-logging` namespace.